Source/CheckoutSdkIos.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 /// Add a keyboard toolbar with navigation buttons. Use it to navigate between text fields.
 ///
 /// - parameter textFields: Array of text fields
-public func addKeyboardToolbarNavigation(textFields: [UITextField]) {
+func addKeyboardToolbarNavigation(textFields: [UITextField]) {
```

Comment:
Question : what is the purpose of addKeyboardToolbarNavigation

Reply:
This is a raw function that just lives in the sdk (its not part of any struct or class or whatever). its used to add the toolbar with the "done" button above the software keyboard when typing. Only change I've made here is to no longer make it public.

When Frames' UI is rewritten this whole file should be deleted, but for now this is still required.